### PR TITLE
Converting the string to a SHA256 hash

### PR DESCRIPTION
**Pull request by sohilpandya**
Please review & Merge after #86 

- we need to use some objc-c code so needed to use something called a bridging header which would allow us to use some obcj-c code in swift
- converting the json string to the hash in startTimer
- related to #84, #87

### CODE REVIEW

File: dwyl/time/stringToHash.swift [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+//  stringToHash.swift
+//  dwyl
+//
+//  Created by Sohil Pandya on 22/06/2017.
+//  Copyright © 2017 dwyl. All rights reserved.
+//
+
+import Foundation
+
+func sha256(_ data: Data) -> Data? {
```
Comment (nelsonic) [inline dwyl/time/stringToHash.swift:11]: @sohilpandya this looks good. we're going to need to write a _test_ to check for the output at some point _soon_ but for now this is fine. thanks! 👍

**Review by nelsonic — APPROVED** [review]
@sohilpandya LGTM. 🎉

**Review by iteles — APPROVED** [review]
Thanks for this @sohilpandya, please add Nelson's comment to your list 👍

**Comment by sohilpandya** [comment]
Thanks @iteles, updated the top comment on here https://github.com/dwyl/time-apple-watch-app/issues/84 by adding the testing as a checklist.

**Pull request merged**